pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -681,7 +681,7 @@ func expandDiskImageOffline(imagePath string, size int64) error {
 
 func possibleGuestSize(disk api.Disk) (int64, bool) {
 	if disk.Capacity == nil {
-		log.DefaultLogger().Error("No disk capacity")
+		log.DefaultLogger().Errorf("No disk capacity on %v", disk.Device)
```

Comment:
A more helpful information would be the source. (To be more precise the path to the file or device)

Reply:
@xpivarc how do I test this changes? I can start the build server on my machine, but I can't figure out how to test just the changes I have made to that unit of code.

Reply:
I think the simplest way is to write a small unit test in manager_test.go. IIRC, it should be able to use the local function.

Reply:
Writing a small unit test for any changes you have demands you have a good grasp of the various kubevirt packages(maybe there is a trick to do it, and I just suck at it).

Reply:
well, you asked a way how to check your change. First, I don't think we need the test for upstream, it is a very simple change for a logging message. However, if you want to check it, you could add a unit test  in [manager_test.go](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/manager_test.go)  and call the function and pass a disk with empty `Capacity`. If you focus on that test, you should see login message